cookbooks/bcpc/recipes/mysql.rb

```diff
@@ -196,7 +196,7 @@
 max_connections =
   [
     (mysql_nodes.length * 50 + all_nodes.length * 5),
-    200
+    300
```

Comment:
What raised the MySQL connection count for `test-laptop`? (Though 300's still plenty low I just don't understand what you were hitting.)